front-end/src/components/Hamburger.js

```diff
@@ -11,6 +11,12 @@ const Hamburger = (props) => {
       <Link to="/my-shows/1" id="My Shows" className="menu-item">
         My Shows
       </Link>
+      <Link to="/meet-the-team/1" id="Meet-the-team" className="menu-item">
```

Comment:
Looks good! I don't think the `/1` after the links to Meet the Team or Terms of Service is necessary though--do you mind removing that?

Reply:
true, will edit that, thanks!